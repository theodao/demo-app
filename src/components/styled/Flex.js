import React from "react";
import styled from "styled-components";
import {flex, flexDirection, flexbox, flexBasis, flexWrap, justifyContent, alignItems} from "styled-system";

const Wrap = styled.div(
  {display: "flex", flexDirection: "column"},
  flex,
  flexDirection,
  flexbox,
  flexBasis,
  flexWrap,
  justifyContent,
  alignItems,
);

const Flex = ({children, ...props}) => <Wrap {...props}>                                     {children}</Wrap>;

export default Flex;
