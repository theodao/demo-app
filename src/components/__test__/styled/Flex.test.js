import React from "react";
import renderer from "react-test-renderer";
import Flex from "../../styled/Flex";

it("renders Button", () => {
  const tree = renderer.create(<Flex />).toJSON();
  expect(tree).toMatchSnapshot();
});
