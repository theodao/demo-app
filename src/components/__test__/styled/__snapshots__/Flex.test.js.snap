// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button 1`] = `
<div
  className="sc-AxjAm kmUfgt"
/>
`;
